preduzece/src/main/java/raf/si/racunovodstvo/preduzece/jobs/ObracunZaradeJob.java

```diff
@@ -46,7 +46,9 @@ public void run() {
                 long delay = now.until(nextDate, ChronoUnit.MILLIS);
 
                 try {
-                    obracunZaradeService.makeObracunZarade(Date.from(now.toInstant()));
+                    obracunZaposleniService.makeObracun(Date.from(now.toInstant()));
+                } catch (Exception e) {
+                    e.printStackTrace();
```

Comment:
Da li postoji neki razlog zasto ovde hvatas gresku?

Reply:
jao zaboravio sam da sredim to, stavio sam catch jer mi nije bacao exception pa sam hteo da vidim sta se desava. Mogao bih u tom catchu da bacim exception dalje mozda?